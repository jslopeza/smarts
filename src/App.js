import React, {Component} from "react";

import {
	View,
	Text,
	StyleSheet,
} from "react-native";

const styles = StyleSheet.create({
	container: {
		flex: 1,
		justifyContent: 'center',
		alignItems: 'center',
		backgroundColor: '#F5FCFF',
	},
})

const App = () => (
	<View style={styles.container}>
		<Text>Hello</Text>
	</View>
);

export default App;