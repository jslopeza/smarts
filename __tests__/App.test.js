import "react-native";
import React from "react";
import App from "../src/App";

import renderer from "react-test-renderer";

describe('App', () => {
  it('renders App correctly', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();
  
    expect(tree).toMatchSnapshot();
  });
});
