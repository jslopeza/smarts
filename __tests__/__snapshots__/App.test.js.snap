exports[`App renders App correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center"
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Hello
  </Text>
</View>
`;
